front/app/containers/ProjectsShowPage/SucessModal.tsx

```diff
@@ -41,10 +46,12 @@ const SuccessModal = ({ projectId }: Props) => {
       }, 1500);
     }
 
-    removeSearchParams(['show_modal', 'phase_id']);
+    removeSearchParams(['show_modal', 'phase_id', 'new_idea_id']);
   }, [showModalParam]);
 
   if (!ready) return null;
+  // If there is a newIdeaIdParam, wait for idea to load
+  if (newIdeaIdParam && !idea) return null;
```

Comment:
Minor, we can combine these
```suggestion
    // If there is a newIdeaIdParam, wait for idea to load
    if (!ready || (newIdeaIdParam && !idea)) return null;
```